src/components/FermentationRecipeList/FermentationRecipeList.test.js

```diff
@@ -62,4 +62,60 @@ describe('FermentationRecipeList', () => {
 
     expect(header).toBeInTheDocument()
   })
+
+  test('filters recipes based on search query', () => {
+    const levelEntitlements = getLevelEntitlements(100)
+    const cropsAvailableToFerment = getCropsAvailableToFerment(
+      levelEntitlements
+    )
+
+    render(<FermentationRecipeListStub levelEntitlements={levelEntitlements} />)
+
+    const searchBar = screen.getByPlaceholderText(
+      'Search fermentation recipes...'
+    )
+    expect(searchBar).toBeInTheDocument()
+
+    fireEvent.change(searchBar, { target: { value: 'apple' } })
```

Comment:
if possible, we should use `userEvent.type` instead of `fireEvent`. note: it requires an `await`. docs here https://github.com/testing-library/user-event (i'm only going to leave this comment here, but i see `fireEvent` in some other tests that i think could be updated as well)